.github/workflows/windows-ci.yaml

```diff
@@ -23,3 +23,7 @@ jobs:
 
       - run: python setup.py bdist_wheel
 
+      - uses: actions/upload-artifact@v2
+        with:
+          name: windows-wheel
+          path: dist/*.whl
```

Comment:
I don't see a good reason why we need to save the wheel either. Is not like they may be useful to reproduce a test error as no tests are run on the CI anyway.

Reply:
What orginal `appveyor.yml` does, too.